### DISCUSSION TITLE
有些非bug问题欢迎来到讨论区分享同时也欢迎讨论效率提升的vim技巧

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [HUAHUAI23/nvim64#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mirroez** [comment]
hi，我使用打包版在linux离线服务器上解压使用，但是运行时报错：
        ./nvim: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by ./nvim) 
        ./nvim: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./nvim) 
似乎是服务器的GLIBC版本过低，我尝试了neovim的版本，发现最高可以支持到neovim 7.2 ,
我在打包版的解压目录下替换neovim为7.2版本，可以打开软件了，但是会报很多错误。有没有什么好的解决办法呢？

**Reply by HUAHUAI23** [reply]
打包版本的好久没更新了，你可以下载那个tar包，然后执行packer sync等命令，下载所有的lsp，已经其他插件，然后将程序的根目录打个tar包就行了，所有的插件都会被放到程序的/share/nvim/xray 这个目录下，最近一直没看消息，有什么问题，我们继续讨论
